pelicun/uq.py

```diff
@@ -482,21 +463,23 @@ def _get_std_corr_matrix(std_samples: np.ndarray) -> np.ndarray | None:
     # otherwise, we can try to fix the matrix using SVD
     except np.linalg.LinAlgError:
         try:
-            U, s, _ = svd(
+            u_matrix, s_matrix, _ = svd(
```

Comment:
`s_matrix` is actually a vector of singular values. So, let's call it `s_vector` for clarity.

Reply:
Right, thanks. 541e0665